session_analyzer.py

```diff
@@ -13,6 +14,7 @@ def __init__(self):
     @asyncio.coroutine
     def analyze(self):
         while True:
+            time.sleep(1)
```

Comment:
I'm not sure it's right way, I experimented with async sleep and I have some problem with it, maybe I should rewrite this part of code and use async sleep instead of time sleep.